src/software/parameter/config/network.yaml

```diff
@@ -12,3 +12,26 @@ NetworkConfig:
       description: >-
           The network interface that is connected to the thunderbots router.
           Can be found using ifconfig on ubuntu.
+    VisionIP:
+      default: "224.5.23.2"
+      type: "std::string"
+      description: The IP address to listen on for vision data
+    VisionPort:
+      min: 1
+      max: 65535
+      default: 10020
+      type: "int"
+      description: The port to listen on for vision data
+    GamecontrollerIP:
+      default: "224.5.23.1"
+      type: "std::string"
+      description: The IP address to listen on for gamecontroller data
```

Comment:
And likewise, if changing to `GamecontrollerIPv4Address`, change the description to specify "IPv4 address"

Reply:
updated descriptions for both vision and gamecontroller